index.js

```diff
@@ -1,16 +1,13 @@
 'use strict';
 
-const differenceWith = require('lodash/differenceWith');
-const isMatch = require('lodash/isMatch');
-const isPlainObject = require('lodash/isPlainObject');
-const isEqualWith = require('lodash/isEqualWith');
+const _ = require('lodash');
```

Comment:
Instead of requiring the whole lodash, could you use destructuring?

```js
const { isPlainObject, isMatch, ... } from 'lodash';
```

Reply:
Sure. What was I thinking, lol...